src/scenes/ModulesRouter/ModuleEventSessionView.tsx

```diff
@@ -53,34 +51,22 @@ const ModuleEventSessionView = () => {
     [activeLocale, moduleId, projectId]
   );
 
-  const eventIsStartingSoon = useMemo(() => {
-    if (session?.startTime) {
-      const startTime = new Date(session.startTime);
-      const diff = startTime.getTime() - now.getTime();
-      return diff < FIFTEEN_MINUTE_INTERVAL_MS;
+  const getEventStatus = () => {
```

Comment:
Can you add this to `src/types/Module.ts`? We have a similar function for `getEventType`